server/tests/place_test.py

```diff
@@ -96,6 +101,11 @@ def test_place(self, mock_get_place_type, mock_get_i18n_name):
         assert response.status_code == 200
```

Comment:
does this need follow_redirects=True? (likewise for the other test that still uses the topic param)

Reply:
Done